.github/workflows/uuid-test.yml

```diff
@@ -0,0 +1,61 @@
+name: Build-Test
```

Comment:
I never deleted this, you might want to delete it unless you have a need to run this workflow locally, and if you do might be better off adding `workflow-dispatch` to the build-test.yml file.